application/F3DStarter.h

```diff
@@ -23,7 +23,7 @@ class F3DStarter
   /**
    * Add a file or directory to the list of paths
    */
-  void AddFile(const std::filesystem::path& path, bool quiet = false);
+  int AddFile(const std::filesystem::path& path, bool quiet = false);
```

Comment:
I think this returned index could be checked and potentially used whenever AddFile is used.

Reply:
the other uses I can see are the "Down" action:
https://github.com/f3d-app/f3d/blob/82aedc9d31603fa52dac5ef3ae50d286ecba9465/application/F3DStarter.cxx#L162-L175

and the initial loading from the CLI
https://github.com/f3d-app/f3d/blob/82aedc9d31603fa52dac5ef3ae50d286ecba9465/application/F3DStarter.cxx#L217-L223

It would make sense to leave the initial case as is, ie. display the first file and have the other ones queued up.

For the down action I don't know, I'm unclear on the use-cases/semantics.

Reply:
I think you are right :)